samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -217,6 +235,7 @@ def cli(
     )  # pragma: no cover
 
 
+# pylint: disable=too-many-branches
```

Comment:
Should we take this opportunity to organize the codebase a bit more?

Reply:
Good catch! Will scan the codebase one more time